ignite/metrics/accuracy.py

```diff
@@ -43,8 +43,10 @@ def _check_shape(self, output: Sequence[torch.Tensor]) -> None:
         if not (y_shape == y_pred_shape):
             raise ValueError("y and y_pred must have compatible shapes.")
 
-        if self._is_multilabel and not (y.shape == y_pred.shape and y.ndimension() > 1 and y.shape[1] != 1):
-            raise ValueError("y and y_pred must have same shape of (batch_size, num_categories, ...).")
+        if self._is_multilabel and not (y.shape == y_pred.shape and y.ndimension() > 1 and y.shape[1] > 1):
+            raise ValueError(
+                "y and y_pred must have same shape of (batch_size, num_categories, ...) " "and num_categories > 1."
```

Comment:
Maybe merge two strings and let `black` do the correct format?

Reply:
Done!